neurodamus/core/configuration.py

```diff
@@ -1106,7 +1107,7 @@ def parse_gid_token(token):
         return gids
 
 
-def check_connections_configure(SimConfig, target_manager):
+def check_connections_configure(sim_config, target_manager):  # noqa: C901, PLR0912, PLR0915
```

Comment:
I understand now why you changed from SimConfig to sim_config: no shadowing. However, SimConfig is a singleton. I think we can remove sim_config alltogether and just refer to SimConfig. 

Please correct me if sim_config is indeed something else

Reply:
open an issue if you are not happy with global state classes

Reply:
> check_connections_configure(config: _SimConfig, target_manager)

I changed it to that.

> open an issue if you are not happy with global state classes

I'll do it if it seems worthwhile.  In this case it does not make sense, since the singleton is created in this file.

Reply:
I leave this open because there is also a question from Weina

Reply:
I am happy with the change, and closing this discussion.